python/sglang/bench_serving.py

```diff
@@ -869,6 +871,18 @@ async def benchmark(
 
     time.sleep(1.5)
 
+    if profile:
+        print("Starting profiler...")
+        profile_input = RequestFuncInput(model=model_id,
+                                        prompt=test_prompt,
+                                        api_url=base_url + "/start_profile",
+                                        prompt_len=test_prompt_len,
+                                        output_len=test_output_len,
+                                        extra_request_body=extra_request_body)
+        profile_output = await request_func(request_func_input=profile_input)
```

Comment:
Do not reuse `request_func`. Many arguments, such as `prompt_len` and `output_len`, are meaningless or incorrect for the `start_profile` endpoint. Can you create a standalone function for this?

Reply:
Thanks for review, the `request_func` function has been replaced with a standalone function.